cloud_sql/sqlserver_instance_pitr/main.tf

```diff
@@ -42,7 +42,7 @@ resource "google_sql_database_instance" "enterprise" {
   region           = "asia-northeast1"
   database_version = "SQLSERVER_2019_ENTERPRISE"
   settings {
-    tier    = "db-custom-4-26624"
```

Comment:
This sample should be for Enterprise, not Enterprise Plus. With this in mind, I think we want to reverse the change on line 45.